test/model.test.js

```diff
@@ -7562,6 +7562,70 @@ describe('Model', function() {
       assert.ok(err == null);
 
     });
+    it('Using bulkSave should not trigger an error (gh-11071)', async function() {
+
+      class TestModel {
+        static _instance
+        _model;
+    
+        /**
+         * @private
+         */
+        constructor() {}
+    
+        static create() {
+            if (this._instance) {return this._instance;}
```

Comment:
Please simplify this test case:

1) Remove the class and methods. Just inline all the method calls.
2) Remove the `try/catch` around `insertMany()`